readme.md

```diff
@@ -63,13 +17,3 @@ client.track({
   userId: 'user id'
 });
 ```
-
-
-## Documentation
-
-Documentation is available at [https://segment.com/libraries/node](https://segment.com/libraries/node).
-
-
-## License
-
-Copyright &copy; 2017 Segment Inc. \<friends@segment.com\>
```

Comment:
Ahps! Segment license? Are we allowed to use this? @paddie

Reply:
yes, but we are not allowed to change the license, see [here](https://github.com/segmentio/analytics-node/blob/master/license)

Reply:
Ok so this is back. I guess remove License is the same as change it?

Reply:
yes :)